mpa/modules/ov/models/mmdet/roi_heads/roi_extractors/single_level_roi_extractor.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import torch
+from mmdet.models.builder import ROI_EXTRACTORS
+from mmdet.models.roi_heads.roi_extractors.single_level_roi_extractor import (
+    SingleRoIExtractor as OriginSingleRoIExtractor,
+)
+from torch import nn
+from torch.nn import functional as F
+from torch.nn.modules.utils import _pair
+
+
+@ROI_EXTRACTORS.register_module()
+class SingleRoIExtractor(OriginSingleRoIExtractor):
```

Comment:
This class also conflicts with Extractor registered in the existing registry.

Reply:
Since this class merely adds negligible features, I believe `force=True` would be enough.
If you insisted to rename it, I would. Let me know.